src/main/controllers/AddUserRolesController.ts

```diff
@@ -30,12 +37,14 @@ export class AddUserRolesController extends RootController {
     }
 
     const roles = fields.roles;
-    await req.scope.cradle.api.registerUser({
-      email: fields._email,
-      firstName: fields._forename,
-      lastName: fields._surname,
-      roles: convertToArray(roles)
-    });
+
+    await this.inviteService.inviteUser(
+      fields._email,
+      fields._forename,
+      fields._surname,
+      convertToArray(roles),
+      req.session.user.id
```

Comment:
you'll need to set a success redirect here or the user won't go anywhere after activation

Reply:
Where should we redirect users who have been invited through the dashboard? We don't know what service they're being invited for here (as it's just name, email and roles) so I'm not sure how the redirect should be used here

Reply:
I think the old endpoints would have defaulted to idam-user-dashboard being the client, so the redirect would be the dashboard one. I think it's fine to do that here - might be interesting to have a new requirement to select the "destination" service for someone being invited.

Reply:
TODO:
- Redirect new support users to dashboard
- Redirect new professional users to gov.uk

Reply:
Addressed in https://github.com/hmcts/idam-user-dashboard/pull/440/commits/a9f590b9b2832e7369095b0d410ac293fd2a4f65